src/codegen/buffering_consumer.cpp

```diff
@@ -40,6 +40,15 @@ WrappedTuple &WrappedTuple::operator=(const WrappedTuple &o) {
   return *this;
 }
 
+std::string WrappedTuple::ToCSV() const {
+  std::string ret;
+  for (uint32_t i = 0; i < tuple_.size(); i++) {
+    if (i != 0) ret.append(",");
+    ret.append(tuple_[i].ToString());
```

Comment:
The problem with this is that we are not escaping commas. Is there a CSV library that we could use?

Reply:
This is just a utility function to pretty print tuples for debugging purposes.  It isn't used in execution.